### DISCUSSION TITLE
OrangePi CM4?

### DISCUSSION THREAD

**Discussion by brauhausdc** [Joshua-Riek/ubuntu-rockchip#681] [General]
Has this been tried on an OrangePI CM4 module previously? I've got one and the associated base board, and am considering giving it a whirl - just want to know if there's any gotchas?

**Comment by jglathe** [comment]
Hmm looks like they have written a dtb, at least worth a try

**Comment by brauhausdc** [comment]
Gave it a spin - it booted, though threw a number of errors on the way up. What would be needed / what can I do to help get this board onto the "supported" list?

**Comment by costastf** [comment]
I would also be interested in what would be needed to add this board to the supported list.

**Comment by tdesaules** [comment]
Hey guys ! Did someone still trying to get some stuff on the cm4 ?

**Comment by brauhausdc** [comment]
See above -- Mine is still running, and it'd be great if it was on the supported list.

**Reply by tdesaules** [reply]
What image did you use as base ? (didn't try yet, I have some Orange Pi 3B (probably start with them, I see there is an image available) but I also have 15 Orange Pi CM4 so I will definitely try to find a replacement for the Orange Pi Ubuntu image)

**Reply by brauhausdc** [reply]
IIRC, I used the Pi3B image - it worked, but had a few errors.

**Reply by tdesaules** [reply]
Ok, thank you for the info. The hardware doesn’t look so different. I will try both !

**Comment by Cringely** [comment]
I was able to get the 24.04 server opi3b image working on my cm4 using the latest build triggered yesterday without any real hassle. Booting from EMMC now as well.

471 - https://github.com/Joshua-Riek/ubuntu-rockchip/actions/runs/8961274876

**Comment by brauhausdc** [comment]
Installed 24.04 yesterday on my CM4 -- seemed to go ok. Ran upgrades and got to -1020 linux but have now started seeing Kernel Panics after a short bit of run time. Suggestions on how to provide feedback? Or should I roll back to 22.04???

**Comment by hkesoglu** [comment]
Hello everyone,

Has anyone recently tested the latest orange pi 3b images on the Orange Pi CM4 (RK3566)?
I'm particularly interested in:

- Stability after full system upgrades (apt dist-upgrade)

- RKISP (camera/V4L2) and RKNPU compatibility

- DTB/overlay adjustments that were required for peripherals (e.g., OV5647)

- Any kernel panics or regressions observed with recent builds (e.g., 6.x series)

If you've had success or run into issues, I'd really appreciate any insights.

Thanks!
